packages/@angular/cli/utilities/app-utils.ts

```diff
@@ -0,0 +1,13 @@
+import {CliConfig as CliConfigInterface} from '../lib/config/schema';
+
+export function getAppFromConfig(apps: CliConfigInterface['apps'], name: String) {
```

Comment:
as a fallback, if the name can be used as a number (using `name.match(/^[0-9]+$/)`), use `parseInt(name, 10)` to get it. This way users can do `ng build --app=1` and get the first app.